### ISSUE TITLE
Request: Package registry

### ISSUE THREAD

**Issue by DanielVoogsgerd** [epi-project/brane#93]
It would be nice to have some place where all premade packages could be listed. It could serve as an example listing of potential packages, and it could allow people to get set up quickly.

My initial thought would be another repo where all packages reside, and then some build script to create a website frontend for it and host it on GH-pages.

**Comment by Lut99** [comment]
An overview would be nice! But maybe a little redundant, because I don't think there are many packages besides what's in `brane-std` or in Brane's `examples` (besides student packages, but we need to contact them before we can publish it since I promised that last two years). So I think `brane-std`'s README may be sufficient until we ever get some real adoption from users.

If you still want to do it, then I'd just make a page/book in the wiki. Partly because we already have infrastructure for that, but also because then we can nest them under `wiki.enablingpersonalizedinterventions.nl` (since I'm not sure we can easily get another domain name).
